packages/grafana-data/src/types/datasource.ts

```diff
@@ -216,6 +216,7 @@ abstract class DataSourceApi<
     this.type = instanceSettings.type;
     this.meta = instanceSettings.meta;
     this.uid = instanceSettings.uid;
+    this.defaultQuery = {};
```

Comment:
wonder if this should instead be a function, so we can have more context specific default queries..

getDefaultQuery(app: CoreApp): Partial<TQuery> {
} 

That we can more easily support different defaults for Explore or an Alerting query.